config/application.rb

```diff
@@ -307,6 +310,13 @@ def config.database_configuration
     require_dependency 'stylesheet/manager'
     require_dependency 'svg_sprite/svg_sprite'
 
+    # This empty initializer forces the :let_zeitwerk_take_over initializer to run before we load
+    # initializers in config/initializers. This is done because autoloading before Zeitwerk takes
+    # over is deprecated but our initializers do a lot of autoloading.
+    # See https://gitlab.com/gitlab-org/gitlab/issues/197346 for more details
+    initializer :move_initializers, before: :load_config_initializers, after: :let_zeitwerk_take_over do
+    end
```

Comment:
I haven't looked into Zeitwek much but I'm not sure I fully understand why we need to do this. Also passing an empty block seems weird here. If we're autoloading before Zeitwerk takes over, can we fix those spots in our code?